From a026eb4196331a76f9b20f74db5f8e69e882d3bf Mon Sep 17 00:00:00 2001
From: Sean Whitton <spwhitton@spwhitton.name>
Date: Sun, 26 Aug 2018 09:24:13 -0700
Subject: [PATCH 2/3] dgit-maint-debrebase(7): Avoid some uses of =item

Commands for the user to type into their shell should have empty lines
above and below them; this ensures that.  Admittedly while making the
POD source harder to read.

Signed-off-by: Sean Whitton <spwhitton@spwhitton.name>
---
 dgit-maint-debrebase.7.pod | 26 +++++++++++++++++---------
 1 file changed, 17 insertions(+), 9 deletions(-)

diff --git a/dgit-maint-debrebase.7.pod b/dgit-maint-debrebase.7.pod
index 675e2f3..cc393b5 100644
--- a/dgit-maint-debrebase.7.pod
+++ b/dgit-maint-debrebase.7.pod
@@ -629,23 +629,27 @@ The git history made by git-debrebase can seem complicated.
 Here are some suggestions for helpful invocations of gitk and git.
 They can be adapted for other tools like tig, git log, etc.
 
-=over
+History of package in Debian, disregarding history from upstream:
 
-=item History of package in Debian (disregarding history from upstream):
+=over
 
     % gitk --first-parent
 
 In a laundered branch, the delta queue is at the top.
 
-=item History of the packaging (excluding the delta queue)
+=back
+
+History of the packaging, excluding the delta queue:
 
     % gitk :/debian :!/debian/patches
 
-=item Just the delta queue (ie, Debian's changes to upstream):
+Just the delta queue (i.e. Debian's changes to upstream):
 
     % gitk --first-parent -- :/ :!/debian
 
-=item Full history including old versions of the delta queue:
+Full history including old versions of the delta queue:
+
+=over
 
     % gitk --date-order
 
@@ -654,18 +658,22 @@ The "Declare fast forward" commits you see have an older history
 and a newer history as the other.
 --date-order makes gitk show the delta queues in the right order.
 
-=item Show complete diff since the last upload:
+=back
+
+Complete diff since the last upload:
+
+=over
 
     % git diff dgit/dgit/sid..HEAD -- :/ :!/debian/patches
 (Includes changes to upstream files.)
 
-=item Interdiff of delta queue since last upload, if you really want that:
+=back
+
+Interdiff of delta queue since last upload, if you really want it:
 
     % git debrebase make-patches
     % git diff dgit/dgit/sid..HEAD -- debian/patches
 
-=back
-
 Also of course there is
 
     % git debrebase status
-- 
2.11.0

